app/controllers/vendor_api/vendor_api_controller.rb

```diff
@@ -44,6 +45,10 @@ def parameter_invalid(e)
       render json: { errors: [{ error: 'ParameterInvalid', message: e }] }, status: :unprocessable_entity
     end
 
+    def internal_server_error(e)
+      render json: { errors: [{ error: 'InternalServerError', message: e }] }, status: :internal_server_error
```

Comment:
This exception message is probably going to be uninformative, perhaps use a hardcoded string:

```suggestion
      render json: { errors: [{ error: 'InternalServerError', message: 'The server encountered an unexpected condition that prevented it from fulfilling the request' }] }, status: :internal_server_error
```